### ISSUE TITLE
valid-values-resource missing Swagger UI annotations

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#3007]
When visiting the swagger UI at http://<Server>:<port>/swagger-ui.html  our reset apis are now documented with a module, description & a link

However 'valid-values-resource' has recently been added and does not have the required swagger annotations

<img width="1471" alt="Screenshot 2020-04-30 at 15 25 55" src="https://user-images.githubusercontent.com/7292002/80722217-0b569400-8af7-11ea-9a3f-c844f227fe0d.png">

**Comment by mandy-chessell** [comment]
The original problem is fixed but I also notices one of the Admin Resource files had `Administration Services OMAS` as a title.  I have fixed it to be just `Administration Serrvices`.

I have also updated the description at the top of the Swagger page as follows:

<img width="1051" alt="Screenshot 2020-05-19 at 17 36 21" src="https://user-images.githubusercontent.com/12354195/82353397-52012500-99f7-11ea-9bf1-7a00adf9e37b.png">

Finally, I noticed a half completed sentence on the OMAG Server Platform documentation and I have fixed that.
